docs/static/upgrading.asciidoc

```diff
@@ -24,54 +24,57 @@ See the following topics for information about upgrading Logstash:
 convenient for you to upgrade them together, and it is not required to be done
 at the same time as long as Elasticsearch is upgraded first.
 
 You should upgrade in a timely manner to get the performance improvements that
-come with Logstash 7.0, but do so in the way that makes the most sense for your
+come with Logstash 8.0, but do so in the way that makes the most sense for your
 environment.
 
-[float]
-==== When Not to Upgrade
+[discrete]
+==== When not to upgrade
 
-If any Logstash plugin that you require is not compatible with Logstash 7.0, then you should wait until it is ready
+If any Logstash plugin that you require is not compatible with Logstash 8.0, then you should wait until it is ready
 before upgrading.
 
-Although we make great efforts to ensure compatibility, Logstash 7.0 is not completely backwards compatible. As noted
-in the Elastic Stack upgrade guide, Logstash 7.0 should not be upgraded before Elasticsearch 7.0. This is both
-practical and because some Logstash 7.0 plugins may attempt to use features of Elasticsearch 7.0 that did not exist
-in earlier versions. For example, if you attempt to send the 7.x template to a cluster before Elasticsearch 7.0, then 
-all indexing likely fail. If you use your own custom template with Logstash,
-then this issue can be ignored.
+Although we make great efforts to ensure compatibility, Logstash 8.0 is not completely backwards compatible. 
+As noted in the Elastic Stack upgrade guide, you should not upgrade Logstash 8.0 before you upgrade Elasticsearch 8.0. 
+This is both
+practical and because some Logstash 8.0 plugins may attempt to use features of Elasticsearch 8.0 that did not exist
+in earlier versions. 
+
+For example, if you attempt to send the 8.x template to a cluster before
+Elasticsearch 8.0, then  all indexing likely fail. 
+If you use your own custom template with Logstash, then this issue can be ignored.
 
 
 [[upgrading-using-package-managers]]
-=== Upgrading Using Package Managers
+=== Upgrading using package managers
 
 This procedure uses <<package-repositories,package managers>> to upgrade Logstash.
 
 1. Shut down your Logstash pipeline, including any inputs that send events to Logstash.
 2. Using the directions in the <<package-repositories>> section, update your repository
-links to point to the 7.x repositories.
+links to point to the 8.x repositories.
 3. Run the `apt-get upgrade logstash` or `yum update logstash` command as appropriate for your operating system.
 4. Test your configuration file with the `logstash --config.test_and_exit -f <configuration-file>` command. Configuration options for
-some Logstash plugins have changed in the 7.x release.
+some Logstash plugins have changed in the 8.x release.
 5. Restart your Logstash pipeline after you have updated your configuration file.
 
 [[upgrading-using-direct-download]]
-=== Upgrading Using a Direct Download
+=== Upgrading using a direct download
```

Comment:
As a side note, the PR #13684 touches the same list, in case we want to have a unique PR

Reply:
Merged before this PR